boa/src/builtins/object/mod.rs

```diff
@@ -410,6 +410,37 @@ pub fn make_object(_: &Value, args: &[Value], ctx: &mut Interpreter) -> ResultVa
     Ok(object)
 }
 
+
+/// `Object.create( proto, [propertiesObject] )`
+///
+/// Creates a new object from the provided prototype.
+///
+/// More information:
+///  - [ECMAScript reference][spec]
+///  - [MDN documentation][mdn]
+///
+/// [spec]: https://tc39.es/ecma262/#sec-object.create
+/// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
+pub fn create_builtin(_: &Value, args: &[Value], interpreter: &mut Interpreter) -> ResultValue {
+    let __proto__ = args.get(0).cloned().unwrap_or_else(Value::undefined);
+    let properties = args.get(1).cloned().unwrap_or_else(Value::undefined);
+
+    if properties != Value::Undefined {
+        unimplemented!("propertiesObject argument of Object.create")
+    }
```

Comment:
we can just check if there are more than one rather than copying the value:
```suggestion
    if args.len() > 1 {
        unimplemented!("propertiesObject argument of Object.create")
    }
```

Reply:
This would also fail `Object.create({}, undefined)`, which passes in the current implementation? And also cases like: 
![image](https://user-images.githubusercontent.com/19698636/86809258-fa3c7d00-c07b-11ea-80ba-5d3c52f6df9b.png)

Although contrived examples, I think we can leave the current implementation.

Reply:
I see. we can leave it as is for now.